lib/chef/resource.rb

```diff
@@ -1090,25 +1099,23 @@ def should_skip?(action)
     # === Returns
     # <Chef::Resource>:: returns the proper Chef::Resource class
     def self.resource_for_node(short_name, node)
-      require 'chef/resource_resolver'
```

Comment:
I sorta suspect this'll cause a dependency loop somewhere for someone like chef-sugar or chefspec.